src/Kernels.cpp

```diff
@@ -1,6 +1,5 @@
 #include <cmath>
-#include <stdio.h>
-#include <cstddef>
+#include <iostream>
```

Comment:
I don't see any use of <iostream> provided features here - do we need it at all?

Reply:
It's used by `numKernel` on line 117

```
if (t > P.NKR)
{
	fprintf(stderr, "** %lg  %lg  %lg**\n", r2, P.KernelDelta, t);
	ERR_CRITICAL("r too large in NumKernel\n");
}
```

I considered changing it to modern c++ style streams with `<<` pipes, but ultimately decided to address that in a future (and separate) pull request, mostly because this PR was starting to get large, and I didn't want a cstream change to distract from my `default(none)` change above.

Reply:
When we move to iostream we should do it everywhere at once.